Support ignore_keywords flag for word delimiter graph token filter (#59563)
elastic/elasticsearch: docs/reference/analysis/tokenfilters/word-delimiter-graph-tokenfilter.asciidoc

in the output. If `false`, the filter excludes these tokens from the output.
Defaults to `true`.

`ignore_keywords`::
(Optional, boolean)

> If `true`, the filter suppresses processing tokens with
> {lucene-core-javadoc}/org/apache/lucene/analysis/tokenattributes/KeywordAttribute.html#isKeyword()[KeywordAttribute.isKeyword()=true]

Reviewer: ```suggestion
If `true`, the filter suppresses processing tokens with
a `keyword` attribute of `true`.
```
